src/components/soknadsStatus/SoknadsStatus.tsx

```diff
@@ -53,37 +53,41 @@ const SoknadsStatus: React.FC<Props> = ({status, sak, leserData}) => {
                     let saksStatus = statusdetalj.status.replace(/_/g, " ");
                     saksStatus = saksStatus === "FERDIGBEHANDLET" ? "FERDIG BEHANDLET" : saksStatus;
                     const kanVises: boolean = statusdetalj.status !== SaksStatus.IKKE_INNSYN;
+                    const sakIkkeInnsyn = saksStatus === SaksStatusEnum.IKKE_INNSYN;
+                    const sakBehandlesIkke = saksStatus === SaksStatusEnum.BEHANDLES_IKKE;
+                    const soknadBehandlesIkke = status === SoknadsStatusEnum.BEHANDLES_IKKE;
                     return (
                         <div className="status_detalj_panel" key={index}>
                             <div className={"status_detalj_linje"}>
                                 <div className="status_detalj_panel__tittel">
                                     <Element>{statusdetalj.tittel}</Element>
                                 </div>
-                                {kanVises &&
-                                    saksStatus !== SaksStatusEnum.IKKE_INNSYN &&
-                                    saksStatus !== SaksStatusEnum.BEHANDLES_IKKE &&
-                                    status !== SoknadsStatusEnum.BEHANDLES_IKKE && (
-                                        <div className="status_detalj_panel__status">
-                                            <EtikettLiten>{saksStatus}</EtikettLiten>
-                                        </div>
-                                    )}
+                                {kanVises && !sakIkkeInnsyn && !sakBehandlesIkke && !soknadBehandlesIkke && (
+                                    <div className="status_detalj_panel__status">
+                                        <EtikettLiten>{saksStatus}</EtikettLiten>
+                                    </div>
+                                )}
                             </div>
-
                             {statusdetalj.melding && statusdetalj.melding.length > 0 && (
                                 <div className="panel-glippe-over">
                                     <Normaltekst>{statusdetalj.melding}</Normaltekst>
                                 </div>
                             )}
-                            {(saksStatus === SaksStatusEnum.BEHANDLES_IKKE ||
-                                status === SoknadsStatusEnum.BEHANDLES_IKKE) &&
-                                saksStatus !== SaksStatusEnum.IKKE_INNSYN && (
-                                    <div className="panel-glippe-over">
-                                        <Normaltekst>
-                                            <FormattedMessage id="status.behandles_ikke_ingress" />
-                                        </Normaltekst>
-                                    </div>
-                                )}
-                            {saksStatus === SaksStatusEnum.IKKE_INNSYN && (
+                            {sakIkkeInnsyn && soknadBehandlesIkke && (
```

Comment:
Jeg forstår ikke helt. Jeg trodde denne teksten skulle komme når soknadBehandlesIkke uavhengig av statusen til saken? Så sjekken kun skal være:
`soknadBehandlesIkke` ? 

Altså at: 
```
soknadBehandlesIkke  --> soknad_behandles_ikke_ingress
sakBehandlesIkke && !soknadBehandlesIkke  --> sak_behandles_ikke_ingress
sakIkkeInnsyn && !soknadBehandlesIkke --> ikke_innsyn_ingress
```

Eller er det noe jeg har misforstått?

Når søknaden behandles ikke. Skal det da stå "søknaden behandles ikke... " under hver eneste sak eller "saken behandles ikke..."? 🙂

Reply:
Hmm, jeg trudde at når en søknad har status `soknadBehandlesIkke` og det ligger en sak som har status `sakIkkeInnsyn` så skulle bare teksten for `soknadBehandlesIkke` vises.

Det var sånn jeg forsto https://jira.adeo.no/browse/DIGISOS-1768. Har jeg misforstått oppgaven?